content/blog/2020-10-20-react-v17.md

```diff
@@ -5,15 +5,15 @@ author: [gaearon,rachelnabors]
 
 Today, we are releasing React 17! We've written at length about the role of the React 17 release and the changes it contains in [the React 17 RC blog post](/blog/2020/08/10/react-v17-rc.html). This post is a brief summary of it, so if you've already read the RC post, you can skip this one.
 
-## No New Features
+## No New Features {#no-new-features}
```

Comment:
The anchors are working on master as well: https://reactjs.org/blog/2020/10/20/react-v17.html#no-new-features

What problems do explicit markers solve here?

Reply:
Hey @eps1lon, I noticed this and actually it seems like those changes would just get added on whenever I ran: 
`yarn check-all`

I left it thinking maybe that's part of the linter, but not worries can remove

Reply:
>  I noticed this and actually it seems like those changes would just get added on whenever I ran:
yarn check-all

Thanks! It's part of `yarn generate-ids`. I included the change while we're at it.

Reply:
Awesome, thank you!